ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3824,10 +3868,11 @@ static int zend_jit_context_threaded_call(dasm_State **Dst, const zend_op *oplin
 {
 	if (!zend_jit_handler(Dst, opline, 1)) return 0;
 	if (opline->opcode == ZEND_DO_UCALL) {
+		|	ENDBR
 		|	call ->context_threaded_call
 	} else {
 		const zend_op *next_opline = opline + 1;
-
+		|	ENDBR
```

Comment:
Better to remove this. Now, I'm not sure if they may be necessary or not.
**context threading** is an experimental mechanism, but it never worked well.

Reply:
Removed.